bhyve.pyx

```diff
@@ -1,4 +1,139 @@
-cimport bhyve
+cimport vmmapi
+cimport cpuset
+cimport types
+cimport vmm
 
+import enum
+import os
 
+from libc.errno cimport errno
+from libc.stdlib cimport free
 
+
+class Error(enum.IntEnum):
+    OPEN_FAILED = 100
+    STATS_FETCH_FAILED = 101
+    RETRIEVE_ACTIVE_VCPU_FAILED = 102
+
+
+class BhyveException(RuntimeError):
+    def __init__(self, code, message):
+        super(BhyveException, self).__init__(f'[Errorno {code}] {message}')
+        self.code = code
+
+cdef class VM:
+
+    cdef str vm_name
+    cdef vmmapi.vmctx* vm
```

Comment:
C pointer must always be declared as ```type *variable```, here it doesn't create any issue, some people declare like ```type * variable``` others ```type* variable```, but on FreeBSD we always use "type *variable".